andela/codango
# #115155919 Display Online Presence

#### What does this PR do?
Displays participants status as online or offline during a pair programming session

#### Description of Task to be completed?
- Add online presence when in a pair session
- Set user to offline when user logs out of the session
- Delete pair session data from firebase when session is deleted

#### How should this be manually tested?
Get to the review app [here](https://codango-staging-pr-135.herokuapp.com/). Navigate to the `Pair Programming page` create a Session, invite others and see when they come online or go offline

#### Any background context you want to provide?
This branch is dependent on features from bg-live-pair-programming-119257909 branch which is yet to be merged to `develop branch`
#### What are the relevant pivotal tracker stories?
#115155919
#### Screenshots (if appropriate)
![screen shot 2016-07-20 at 6 03 06 pm](https://cloud.githubusercontent.com/assets/17270426/17004978/a26e2a82-4ecf-11e6-8a30-b44cb666a01a.png)
